### PR TITLE
Always show the brightness slider in more-info

### PR DESCRIPTION
**Pull request by andrey-git**
Always show the brightness slider in more-info if it is supported.
Even when the light is currently off.

Implements (1) of this: https://community.home-assistant.io/t/ui-more-suited-for-dimmers/8660/3

light off
cold_start

Moving the slider turns the light on to set brightness
turned_onn

### CODE REVIEW

File: src/more-infos/more-info-light.html [hunk 1]
```diff
@@ -140,9 +140,15 @@
...
 
   computeClassNames: function (stateObj) {
-    return window.hassUtil.attributeClassNames(
-      stateObj, ['brightness', 'rgb_color', 'color_temp', 'white_value',
+    var classes = window.hassUtil.attributeClassNames(
+      stateObj, ['rgb_color', 'color_temp', 'white_value',
         'effect_list']);
+    // If brightness is supported - show the slider even if the attribute is
+    // missing (because the light is off).
+    if (stateObj && (stateObj.attributes.supported_features & 1)) {
```
Comment (balloob) [inline src/more-infos/more-info-light.html]: Can you store `1` as a constant so we know what the number represents?

**Comment by Maxr1998** [comment]
This change seems to have broken brightness slider for me. It doesn't show at all on the latest dev version. I didn't change my config in between. Adding `has-brightness` to the child `<div>` of `<more-info-light>` manually in Chrome shows the slider temporarily, but after changing the brightness it gets hidden again.

state-off
state-on
state-with-brightness

**Comment by andrey-git** [comment]
@Maxr1998 What kind of dimmer is it?
Could you paste here its full attribute list?
I'm specifically interested in supported_features

**Comment by Maxr1998** [comment]
It's a MQTT JSON light. This is from my lights.yaml:
```yaml
- platform: mqtt_json
  name: "Deckenlicht Max"
  state_topic: "max/light/main/status"
  command_topic: "max/light/main/set"
  brightness: true
```

**Comment by andrey-git** [comment]
`mqtt_json` is missing the supported_features declaration.

Sent home-assistant#5250

**Comment by Maxr1998** [comment]
Ah! Thanks for the fix! :)

**Pull request merged**